src/Metadata/ClassMetadata.php

```diff
@@ -126,6 +126,11 @@ class ClassMetadata extends MergeableClassMetadata
      */
     public $xmlDiscriminatorNamespace;
 
+    /**
+     * @var string
+     */
+    public $excludeIf;
```

Comment:
Can it be also Expression?

Reply:
It should... I've taken over this part from the PropertyMetaData. I will also change it in the PropertyMetadata.